mlrun/model_monitoring/controller.py

```diff
@@ -41,173 +39,174 @@
 )
 from mlrun.utils import datetime_now, logger
 
+_UTF_8 = "utf-8"
+_SECONDS_IN_DAY = int(datetime.timedelta(days=1).total_seconds())
+
+
+# `Schedules` is a dictionary of registered application name as key and Unix timestamp as value
+_Schedules = NewType("_Schedules", dict[str, int])
```

Comment:
Can we create a `Schedules` class that get the json file and have few method like `get(), update(), update_db(), delete()` and some more maybe... I just think it can be more user friendly for us. And can be used also from  MEP creation/deletion API.

Reply:
I think that I get what you mean, but this is basically the `DataItem` API we use here, isn't it?
I'll provide some more abstraction to reduce the duplication between MEP creation/deletion and the controller.